### PR TITLE
update build.psm1 to support packaging for opensuse 13.2

### PR DESCRIPTION
**Pull request by TravisEz13**
@PowerShell/area-build 
Addresses part of #2393 
I have a docker container I build with the package I manually built with this here:  https://hub.docker.com/r/travisez13/powershell/tags/

### CODE REVIEW

**Comment by iSazonov** [comment]
Whether we need to update https://github.com/PowerShell/PowerShell/blob/master/docs/installation/linux.md ?

**Comment by TravisEz13** [comment]
I plan on updating the instructions after we have a package.  There is a separate step in the issue driving this work to update the `linux.md`.

File: build.psm1 [hunk 1]
```diff
@@ -973,10 +975,17 @@ function Install-Dotnet {
 }
 
 function Get-RedHatPackageManager {
+    param(
+        [Parameter(Mandatory=$true)]
+        [switch]
+        $install
```
Comment (daxian-dbw) [inline build.psm1]: I don't see this parameter is used in `Get-RedHatPackageManager`

File: build.psm1 [hunk 2]
```diff
@@ -973,10 +975,17 @@ function Install-Dotnet {
...
+    )
     if ($IsRedHatFamily -and $IsCentOS) {
-        "yum"
+        "yum install -y -q"
     } elseif ($IsRedHatFamily -and $IsFedora) {
```
Comment (daxian-dbw) [inline build.psm1]: `$IsRedHatFamily` seems redundant in `if` and `elseif`

File: build.psm1 [hunk 3]
```diff
@@ -1047,11 +1056,21 @@ function Start-PSBootstrap {
             # Packaging tools
             if ($Package) { $Deps += "ruby-devel", "rpm-build", "groff" }
 
-            $PackageManager = Get-RedHatPackageManager
+            $PackageManager = Get-RedHatPackageManager -install
+
+            $currentUser = (Get-Process -Id $pid -IncludeUserName).UserName
```
Comment (daxian-dbw) [inline build.psm1]: maybe use `id -u` and compare the return user id with 0 to determine if it's running as root.

File: build.psm1 [hunk 4]
```diff
@@ -1047,11 +1056,21 @@ function Start-PSBootstrap {
             # Packaging tools
             if ($Package) { $Deps += "ruby-devel", "rpm-build", "groff" }
 
-            $PackageManager = Get-RedHatPackageManager
+            $PackageManager = Get-RedHatPackageManager -install
```
Comment (daxian-dbw) [inline build.psm1]: same here. `-Install` doesn't do anything.

File: build.psm1 [hunk 5]
```diff
@@ -1622,6 +1651,9 @@ esac
     } elseif ($IsFedora) {
         $version_id = $LinuxInfo.VERSION_ID
         $rpm_dist = "fedora.$version_id"
+    } elseif ($IsOpenSUSE13) {
```
Comment (daxian-dbw) [inline build.psm1]: The comment right above this if block is
> We currently only support CentOS 7 and Fedora 24+

It should be updated with SUSE.

**Review by daxian-dbw — CHANGES_REQUESTED** [review]

File: build.psm1 (hunk 1 above)
Comment (TravisEz13) [inline build.psm1]: Right now we are only getting an install command.  
I can remove it if you think that is a better alternative.

File: build.psm1 (hunk 2 above)
Comment (TravisEz13) [inline build.psm1]: Existing code, but fixed in both places.

File: build.psm1 (hunk 1 above)
Comment (daxian-dbw) [inline build.psm1]: I think it's better to make each check-in self-contained. I suggest to remove this parameter in this PR, and you can add it back in the next PR when the installation logic is moved here.

File: build.psm1 (hunk 3 above)
Comment (TravisEz13) [inline build.psm1]: using existing switch on cmdlet instead.

**Review by daxian-dbw — APPROVED** [review]

**Pull request merged**